pydantic_ai_slim/pydantic_ai/durable_exec/dbos/_agent.py

```diff
@@ -56,6 +57,7 @@ def __init__(
             event_stream_handler: Optional event stream handler to use instead of the one set on the wrapped agent.
             mcp_step_config: The base DBOS step config to use for MCP server steps. If no config is provided, use the default settings of DBOS.
             model_step_config: The DBOS step config to use for model request steps. If no config is provided, use the default settings of DBOS.
+            fastmcp_toolset_step_config: The base DBOS step config to use for FastMCP toolset steps. If no config is provided, use the default settings of DBOS.
```

Comment:
How would you feel about using the existing `mcp_step_config`?

Reply:
Sounds good - I can update this.